Bottom: fafc223b20ab926a790b7793dff4cd0102c5d2ac
Top:    9370d714acf10e98c5a268b64b98710e69e2642e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:31:43 -0500

Fix credential copy to only run on daedalus

Credentials for all hosts end up on daedalus, so limit that play.


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 7cd187c5..9ca28fa2 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -119,7 +119,7 @@
     assemble_scripts: true
 
 - name: Cleanup tasks
-  hosts: all
+  hosts: daedalus
   remote_user: ansibleremote
   become: true
   become_user: root
@@ -127,14 +127,7 @@
   tasks:
   - name: Copy credentials to the /etc dir
     copy:
-      src: ./credentials/{{ inventory_hostname }}
-      dest: /etc/ansible-credentials/
-      owner: root
-      group: root
-      mode: 'u=rw,g=,o='
-  - name: Copy build system credentials to /etc dir
-    copy:
-      src: ./credentials/buildsystem.novatech-llc.com
+      src: "{{ playbook_dir }}/credentials/"
       dest: /etc/ansible-credentials/
       owner: root
       group: root
